software/19.Socket.io/README.md

```diff
@@ -0,0 +1,131 @@
+# Workshop 19 - Chat with Socket.IO
+
+✔ Understand the sockets
+
+## Step 0 : Setup
+
+Any information about the setting of this workshop are available in [SETUP.md](./SETUP.md).
+
+Be careful to finish every step before moving one to the next thing.
+
+A cheatsheet based on Typescript is available [here](https://github.com/PoCInnovation/Workshops/blob/master/software/8.Typescript/HELP.md).
+
+[What's a socket ?](https://socket.io/docs/v4/index.html)
+
+## Step 01 :  First step, Hello World !
+
+If the installation has been managed correctly, you can run your server with
+```bash
+ npm run dev
+```
+
+If you go on your browser at http://localhost:8080/
+you should see this on the screen :
+
+```
+Hello World !
+```
+The server launch and don't need to be launch again with nodemon.
+The script in your package.json allow to take account of modifications
+of your code to launch the server by itself.
+
+## Step 02 :
+
+Your server work with express, you can find it in
+src/index.ts.
+You need to change this file to allow express and socket.io to use it at the same time.
+That will allow you to issue and receive data on your server.
+
+Once the server is set, we will determine the path of our html page on our route '/' that you can find in router/router.ts.
```

Comment:
Link to file.
Where is the HTML page? Should we define a new resolver in our express server? Which method?